.github/workflows/tox.yml

```diff
@@ -1,95 +0,0 @@
-name: Tox
```

Comment:
Same comment as above about updating the link to the badge in the `README.md` file

Reply:
Updated as well.